README.md

```diff
@@ -1,4 +1,41 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
+
+# Hongting Yin . Jarvis Consulting
+
+I am a recent graduate of the University of Waterloo with a degree in Master of Engineering and my focus was on software development and data processing when I was pursuing my master's degree. I'm an enthusiastic and goal-oriented engineer with a passion for software and data engineering and project-based experience. I'm also a detail-oriented and dedicated team player with a 'can-do' attitude while teaming collaboration. Now I'm working at Jarvis on projects related to Linux bash and core Java. During my spare time, I enjoy watching movies, traveling, and jogging.
```

Comment:
remove  ~~Now I'm working at Jarvis on projects related to Linux bash and core Java.~~